wiki/en/Running-a-Server.md

```diff
@@ -50,55 +49,47 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
 
 - Running a server may require you to adjust any firewalls running on or outside of your machine or cloud host.
 
-- Running a Private Server at home will require you to [port forward](#running-a-private-server) on your router. When running a Public Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
+- Running an Uregistered Server at home will require you to [port forward](#running-an-unregistered-server) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
 
 - Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and server. There are plans to expand IPv6 support.
 
-
 # Server Types
 
-You can run your server in one of three "modes" (either at home or on a 3rd party host):
-
-### 1. Public
+You can run your server in different ways (either at home or on a 3rd party host):
 
-Use this mode when you want anyone to join your server, whoever they are.   
+### 1. Registered 
 
-Your server will be listed in the Directory that clients use by default. Musicians can then discover and connect to your server.
+Your server will be listed in a Directory. By default, Jamulus has a list of Public Directories that clients can connect to. If you register with one of these, anyone can then discover and connect to your server. 
 
 <figure>
-	<img src="{% include img/en-screenshots/diagram-public-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Public Server">
-<figcaption>How Public Servers work</figcaption>
+	<img src="{% include img/en-screenshots/diagram-reg-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Registered Server">
+<figcaption>How Registered Servers work</figcaption>
 </figure>
 
+### 2. Unregistered 
 
-
-### 2. Private
-
-This is the default when starting a server for the first time. Private Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful because Jamulus does not (in any server mode) let you control who can connect to a server.
-
+This is the default when starting a server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful because Jamulus does not let you control who can connect to a server.
```

Comment:
> This is the default when starting a server for the first time.

Not yet :).  By default - if running with the UI - you're registered with Any Genre 1.  If you run headless, you'll not register unless you supply a directory address (because headless servers don't have inifiles, not supplying a directory is the only way to say "don't register me with Any Genre 1").

Reply:
(I can't remember if I kept this existing behaviour -- I think I did...  It's probably worth including the change if I didn't already...)

Reply:
Do we not intend to make it so that the default with the GUI is unregistered?

Reply:
Yep, but my PR as it stands hasn't changed legacy behaviour.  I'd also need to review the messing about when the settings are loaded to make sure everything still worked as intended.

I can do it in https://github.com/jamulussoftware/jamulus/pull/2199 ... probably won't be today.

Reply:
Jamulus 3.8.1 GUI server on Windows registers by default with Any Genre 1.

And... another new profile...
![image](https://user-images.githubusercontent.com/1549463/148696477-49567b99-aaa6-4f11-ab59-d5c6b4a52081.png) 
![image](https://user-images.githubusercontent.com/1549463/148696490-632e9c87-c99f-4333-bbef-b14c8ad95667.png)

**However**... now I check, Jamulus _master_ (ff281ca1 2022-01-08) on Linux doesn't...

So I take this all back, I must have done it in one of the "OCD re-order" changes as a "freebie add-on".  Or accident.